### ISSUE TITLE
Django Girls Workshop 19.2.2017

### ISSUE THREAD

**Issue by sfkeller** [SwissPy/sps17#29]
Hi there, it seems that the Django Girls Workshop can take place at Saturday, 19.2.2017...
I'm mainly opening this issue in order to inform you.
These are following possible collaboration (sub-)issues:
* Help outreach (advertising) by pointing to Django Girls Workshop 
* Ask registrants to help out as coach
* ...

**Comment by href** [comment]
Let's get this going :)

This is what we need in any case:

- [x] Mention it on twitter.
- [x] Add a paragraph about it on the website.

I will add this to the website and mention it on Twitter over the weekend.

I'm not sure yet we need to reach out to our attendees. Are more coaches needed?

If so we have multiple options:

- Let everyone know that have already registered.
- Mention it in the e-mail to new registrants
- Mention it in the form for the registrants (they could just mark a checkbox or something)

I'll ping the DjangoGirls organisers to participate in this issue. This also concerns @chiesax and @andrekeller which do the registration.

**Comment by href** [comment]
There's now a mention about the workshop on our website:
http://www.python-summit.ch/#learn-how-to-python-at-the-django-girls-workshop

I've also mentioned it on Twitter. Comment on this ticket if there's more we can help you with.

**Comment by sam-88** [comment]
Hi, thanks a lot and sorry for the late reply! :-)

I just put your logo and a link to your conference website on our website (djangogirls.org/rapperswil).

Also, I will mention your summit on our Twitter profile this week.

Will you send a newsletter/ email to your participants? If yes, it would be great if you could mention our workshop as well. Maybe some of your participants have female friends, spouses, family members etc. that they would like to introduce to Python :-)

We will also send an email to our participants and we could mention your event, if you like. What do you think?

Cheers!

**Comment by chiesax** [comment]
Hi, will this be taught in english?

**Comment by dbrgn** [comment]
To quote the website:

> Wir planen die Einführung, Talks etc in Deutsch zu machen. Das Coaching wird in Deutsch oder Englisch stattfinden. Je nach Coaches können wir sogar auch andere Sprachen anbieten. Wenn du Deutsch also einigermassen verstehst und gut Englisch sprichst solltest du dich in jedem Fall bewerben.

(So both German and English, but knowing some German helps.)

**Comment by chiesax** [comment]
Thx.

**Comment by chiesax** [comment]
@sam-88 we are sending an email to our attendees. Which informations should we provide? Are you looking for coaches?

**Comment by chiesax** [comment]
... and yes feel free to send an email to your participants, thanks a lot!

**Comment by sykaeh** [comment]
Sorry for not commenting earlier. I somehow missed this whole issue. We are currently still looking for a couple of coaches (about 3) and there are still a couple of open spots for participants as well. I do not know what the status is of your communication with the attendees, but if possible it would be great to word out (especially for coaches since the attendees are likely very good candidates for coaches).

**Comment by href** [comment]
We discussed this internally and we want to avoid sending an e-mail to all participants. Our attendees didn't signup for newsletters, except for one sponsor e-mail they'll be sent and we'd like to keep it at that.

We already have you on our website, though it's considerably below the fold. We also added a shout-out on twitter. As far as communication with our participants go I'm afraid this is about what we can offer.
